users/templates/users/user_login.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="row">
+
+        <div class="col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 col-lg-4 col-lg-offset-4">
+
+            <h2>Login</h2>
+
+            {% if messages %}
+                {% for message in messages %}
```

Comment:
Is there no block or something for these dismissable alerts? Seems like this will lead to code duplication when used consistently.

Anyways, nice way to notify the user of any errors. Should be used more often.